ts/features/bonus/cgn/screens/merchants/CgnMerchantDetailScreen.tsx

```diff
@@ -105,85 +105,85 @@ const CgnMerchantDetailScreen: React.FunctionComponent<Props> = (
       }
       contextualHelp={emptyContextualHelp}
     >
-      <SafeAreaView style={IOStyles.flex}>
-        {isReady(merchantDetail) ? (
-          <>
-            <ScrollView style={IOStyles.flex} bounces={false}>
-              {merchantDetail.value.imageUrl && (
-                <View style={{ paddingHorizontal: 16 }}>
-                  <Image
-                    source={{ uri: merchantDetail.value.imageUrl }}
-                    style={styles.merchantImage}
-                  />
-                </View>
-              )}
-              <View style={IOStyles.horizontalContentPadding}>
-                <VSpacer size={24} />
-                <H1>{merchantDetail.value.name}</H1>
-                <VSpacer size={16} />
-                <H2>{I18n.t("bonus.cgn.merchantDetail.title.deals")}</H2>
-                <VSpacer size={8} />
-                <FlatList
-                  data={merchantDetail.value.discounts}
-                  renderItem={renderDiscountListItem}
-                  keyExtractor={(item: Discount) => item.name}
+      {isReady(merchantDetail) ? (
+        <ScrollView style={IOStyles.flex} bounces={false}>
```

Comment:
In my experience, applying `flex: 1` through the `style` property can lead to unexpected side effects (visual glitches, etc...). It's better to add the `flexGrow` style property via `contentContainerStyle`.

Also, disabling the `bounces` prop is not recommended because it might give the citizen the impression that something's wrong. 
```suggestion
        <ScrollView contentContainerStyle={{ flexGrow: 1 }} bounces={true}>
```

Reply:
thanks! adressed in [c1a57b0](https://github.com/pagopa/io-app/pull/4831/commits/c1a57b0c7fb777fdea750d7900d6b0cb07ef0c70), also did a small style refactor to remove surplus elements